nextcord/guild.py

```diff
@@ -1647,6 +1656,9 @@ async def edit(
             The ``icon``, ``banner``, ``splash``, ``discovery_splash``
             parameters now accept :class:`File`, :class:`Attachment`, and :class:`Asset`.
 
+        .. versionadded:: 2.4
```

Comment:
```suggestion
        .. versionchanged:: 2.4
```
The other ... is added are `versionchanged`